asm/istio/istio-operator.yaml

```diff
@@ -11,16 +11,56 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+#
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
-  profile: asm-gcp
+  profile: empty
```

Comment:
Did we remove the asm-gcp profile?

Reply:
It's not removed, it's still in the binary and will still work. This operator isn't going to use it anymore. We had too many cases of the profile setting some option that was unexpected so we're explicitly setting everything ourselves.